README.md

```diff
@@ -37,7 +37,7 @@ Using [npm](https://www.npmjs.com/):
 
 Then with a module bundler like [webpack](https://webpack.github.io/) that supports either CommonJS or ES2015 modules, use as you would anything else:
 
-```js
+```jsx
```

Comment:
I'm kidding too :p

I actually started doing separately using `js` for the non-JSX snippets in a bout of idiocy before I realized how awful that would have been.